api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -920,6 +920,19 @@ public void isIdentifierInUseByAnotherPatient_shouldReturnTrueWhenPatientIdentif
 		PatientIdentifier patientIdentifier = new PatientIdentifier("7TU-8", pit, null);
 		Assert.assertTrue(patientService.isIdentifierInUseByAnotherPatient(patientIdentifier));
 	}
+
+	/**
+	 * @see PatientService#isIdentifierInUseByAnotherPatient(PatientIdentifier)
+	 */
+	@Test
+	@Ignore("Unignore after investigating and fixing - TRUNK-2758")
+	@Verifies(value = "should return false when patient identifier Type is Null", method = "isIdentifierInUseByAnotherPatient(PatientIdentifier)")
+	public void
+	isIdentifierInUseByAnotherPatient_shouldReturnFalseWhenPatientIdentifierTypeIsNull()
+			throws Exception {
```

Comment:
@winglam shouldn't this test be about creating a patient without an identifier type, as the ticket suggests?